modules/rosa-policy-shared-responsibility.adoc

```diff
@@ -4,152 +4,440 @@
+AWS is responsible for protecting the hardware infrastructure that runs all of the services offered in the AWS Cloud. The customer is responsible for incident and operations management of customer application data and any custom networking the customer may have configured for the cluster network or virtual network.
 [cols= "2a,3a,3a",options="header"]
 |===
 
 |Resource
-|Red Hat and AWS responsibilities
+|Service responsibilities
 |Customer responsibilities
 
 |Application networking
-|Monitor cloud load balancers and native OpenShift router service, and respond to alerts.
-|- Monitor health of service load balancer endpoints.
-- Monitor health of application routes, and the endpoints behind them.
-- Report outages to Red Hat.
+|**Red Hat**
+
+- Monitor native OpenShift router
+service, and respond to alerts.
+|- Monitor health of application routes, and the endpoints behind them.
+- Report outages to Red Hat and AWS.
+
+|Virtual networking management
+|**Red Hat**
+
+- Monitor AWS load balancers, Amazon VPC subnets, and AWS service components necessary for default
+platform networking. Respond to alerts.
+|- Monitor health of AWS load balancer endpoints.
+- Monitor network traffic that is optionally configured through Amazon VPC-to-VPC connection, AWS VPN connection, or AWS
+Direct Connect for potential issues or
+security threats.
+
+|Virtual storage management
+|**Red Hat**
+
+- Monitor Amazon EBS volumes used for cluster nodes, and Amazon S3 buckets used for the ROSA service’s built-in container image
+registry. Respond to alerts.
+|- Monitor health of application data.
+- If customer managed AWS KMS keys are
+used, create and control the key lifecycle and
+key policies for Amazon EBS encryption.
+
+|AWS software (public AWS services)
+|**AWS**
+
+- For information regarding AWS incident and operations management, see link:https://docs.aws.amazon.com/whitepapers/latest/aws-operational-resilience/how-aws-maintains-operational-resilience-and-continuity-of-service.html#incident-management[How AWS maintains operational resilience and continuity of service] in the AWS whitepaper.
+|- Monitor health of AWS resources in the
+customer account.
+- Use IAM tools to apply the appropriate
+permissions to AWS resources in the customer account.
+
+|Hardware/AWS global infrastructure
+|**AWS**
 
-|Virtual networking
-|Monitor cloud load balancers, subnets, and public cloud components necessary for default platform networking, and respond to alerts.
-|Monitor network traffic that is optionally configured through VPC to VPC connection, VPN connection, or Direct connection for potential issues or security threats.
+- For information regarding AWS incident and operations management, see link:https://docs.aws.amazon.com/whitepapers/latest/aws-operational-resilience/how-aws-maintains-operational-resilience-and-continuity-of-service.html#incident-management[How AWS maintains operational
```

Comment:
This is in same content with above part, is this expected?

Reply:
Thanks for reviewing these docs, @xueli181114. Yes, this is expected per AWS. -ML